src/server.cpp

```diff
@@ -91,16 +119,23 @@ void GraftServer::setCoapRouters(CoapConnectionManager& coapcm)
 
 void GraftServer::initGlobalContext()
 {
-//  TODO: why context intialized second time here?
-    graft::Context ctx(m_looper->getGcm());
-    const ConfigOpts& copts = m_looper->getCopts();
+//  //TODO: why context intialized second time here?
```

Comment:
clean up  in order

Reply:
What exactly should be done here?
Revert this code to original state? As it was before this commit?
Or remove these commented and useless lines at all?